charts/flyte-core/Chart.yaml

```diff
@@ -3,3 +3,8 @@ name: flyte-core
 description: A Helm chart for Flyte core
 type: application
 version: v0.1.10 # VERSION
+dependencies:
```

Comment:
We can specify the conditional here as opposed to everywhere in the child chart:
https://helm.sh/docs/chart_best_practices/dependencies/#conditions-and-tags

Reply:
Also note how `flyte-binary` uses the `flyteagent` service: https://github.com/flyteorg/flyte/blob/809c6572cc2a62f5b16c9e52ec70b827120c66be/charts/flyte-binary/templates/configmap.yaml#L69

Reply:
good catch, updated it, thanks